src/app/hooks/useOptimizely/index.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import useServerSide from './useServerSide';
+import useClientSide from './useClientSide';
+
+type Props = {
+  experimentName?: string;
```

Comment:
Should this be mandatory?
```suggestion
  experimentName: string;
```